README.md

```diff
@@ -58,6 +58,7 @@ _General-purpose middleware to wrap around ASGI apps._
 - [datasette-auth-github](https://github.com/simonw/datasette-auth-github) - GitHub OAuth authentication for ASGI apps. Supports restricting to specific users or member of specific teams or organizations.
 - [msgpack-asgi](https://github.com/florimondmanca/msgpack-asgi) - Drop-in MessagePack support for ASGI apps and frameworks.
 - [ProxyHeadersMiddleware](https://github.com/encode/uvicorn/blob/master/uvicorn/middleware/proxy_headers.py) - Use `X-Forwarded-Proto` and `X-Forwarded-For` headers set by a known and trusted proxy to make `client` and `scheme` reference the connecting client (shipped with Uvicorn).
+- [Scout APM Starlette integration](https://docs.scoutapm.com/#starlette) - Scout APM (Application Performance Monitoring) integration with Starlette and Starlette-based frameworks. (Part of the `scout-apm` package.)
```

Comment:
Out of consistency with the other commercial entry here (Sentry), let's perhaps stick to a strict mention of the product name?

```suggestion
- [Scout APM Starlette integration](https://docs.scoutapm.com/#starlette) - Scout APM integration with Starlette and Starlette-based frameworks. (Part of the `scout-apm` package.)
```

Reply:
Sure, though APM is a generic term and I wanted to expand it to give some idea what it does. Similarly the Sentry entry isn't clear what Sentry does.

Reply:
Agreed in all fairness, though we'll be able to resolve this more broadly via #25. :+1: